learn/experimental/vector_search.mdx

```diff
@@ -140,6 +140,35 @@ curl -X POST -H 'content-type: application/json' \
 ]'
 ```
 
+### Distribution shift
+
+Semantic search results can sometimes have very close ranking scores. This may result in hybrid searches returning fewer keyword search results than expected. Use `distribution` when configuring an embedder to correct result distribution with an affine transformation:
```

Comment:
From what I've been told, it can also lead to fewer semantic search results.

Reply:
```suggestion
For mathematical reasons, the `_rankingScore` of semantic search results tend to be closely grouped around an average value that depends on the embedder and model used.

This may result in relevant semantic hits being underrepresented and irrelevant semantic hits being overrepresented compared with keyword search hits.

For models that Meilisearch is aware of, that is, all OpenAI models and the Hugging Face model uses by default, Meilisearch already corrects the `_rankingScore` of semantic hits.

For other models, use `distribution` when configuring an embedder to correct the returned `_rankingScore`s of the semantic hits with an affine transformation.

To correctly use `distribution`:

1. First configure your embedder *without* the `distribution` setting
2. index your documents
3. Make several semantic search requests with the `showRankingScore` query parameter set to `true`.
4. Observe the results and manually decide if they are very relevant to your query, somewhat relevant to your query, or not relevant to your query.
5. Check the `_rankingScore`s of the results.
    1. the `_rankingScore`s of the "somewhat relevant" hits should be used as the `mean` of the `distribution` setting.
    2. the average difference between the `_rankingScore` of "very relevant" hits and "somewhat relevant" hits on one hand, and the `_rankingScore` of "somewhat relevant" hits and "not relevant" hists on the other hand should be used as the `sigma` of the `distribution` setting. 
```